enrolments/serializers.py

```diff
@@ -82,6 +82,12 @@ def get_families(self, obj):
             for enrolment in obj.enrolments.filter(active=True)
         ]
 
+    def update(self, instance, validated_data):
+        instance.name = validated_data.get("name", instance.name)
+        instance.attendance = validated_data.get("attendance", instance.attendance)
+        instance.save()
+        return instance
```

Comment:
since id (pk) & families (serializer method field) are both read-only by default, this winds up being the default update behaviour! you might just be able to remove this code

Reply:
:0 very cool, removed